ecosystem.toml

```diff
@@ -54,8 +58,78 @@ repo = "https://github.com/jacklund/repl-rs"
 crates = ["repl-rs"]
 topics = ["repl"]
 
+[[project]]
+name = "cranelift"
+description = "Cranelift is a low-level retargetable code generator. It translates a target-independent intermediate representation into executable machine code."
+repo = "https://github.com/bytecodealliance/wasmtime/tree/main/cranelift"
+crates = ["cranelift-codegen", "cranelift-frontend", "cranelift-jit", "cranelift", "cranelift-module", "cranelift-codegen-meta", "cranelift-wasm", "cranelift-native", "cranelift-reader", "cranelift-object"]
+topics = ["runtime"]
+
+[[project]]
+name = "llvm-sys.rs"
+description = "Rust bindings to LLVM's C API."
+repo = "https://gitlab.com/taricorp/llvm-sys.rs"
+crates = ["llvm-sys"]
+topics = ["runtime"]
+
+[[project]]
+name = "Inkwell"
+description = "It's a New Kind of Wrapper for Exposing LLVM (Safely)"
+repo = "https://github.com/TheDan64/inkwell"
+crates = ["inkwell"]
+topics = ["runtime"]
+
+[[project]]
+name = "HVM"
+description = "A massively parallel, optimal functional runtime in Rust"
+repo = "https://github.com/Kindelia/HVM"
+crates = ["HMV"]
+topics = ["runtime"]
+
 [[showcase]]
 name = "AbleScript"
 description = "AbleScript is a procedural, cursed, dynamically-typed, interpreted programming language, designed to be bad."
 repo = "https://git.ablecorp.us/AbleScript/able-script"
 docs = "https://ablecorp.github.io/able-script-the-book/"
+
+[[showcase]]
+name = "Rust"
```

Comment:
Should we specify rustc? We don't have to, but it might be good

Reply:
Personally I'm agnostic on this one, not sure we need to showcase Rust, but at the same time it doesn't hurt.

Reply:
I think it's kinda cool to mention since it's a really good compiler, and it's written in Rust, but I wouldn't hate not having it. Same thing about rust-analyzer I mentioned in the main comment.

Reply:
This was mostly tongue in cheek so whatever you want honestly 😎

Reply:
Let's leave it for now.